specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -11249,6 +11277,11 @@
           "title": "The command line of the Job Preparation task.",
           "description": "The command line does not run under a shell, and therefore cannot take advantage of shell features such as environment variable expansion. If you want to take advantage of such features, you should invoke the shell in the command line, for example using \"cmd /c MyCommand\" in Windows or \"/bin/sh -c MyCommand\" in Linux."
         },
+        "containerSettings": {
+          "$ref": "#/definitions/TaskContainerSettings",
+          "title": "The settings for the container under which the Job Preparation task runs.",
+          "description": "When this is specified, all directories recursively below the AZ_BATCH_NODE_ROOT_DIR (the root of batch directories on the node) are mapped into the container, all task enviornment variables are mapped into the container, and the task command line is executed in the container."
```

Comment:
Typoed 'environment' again - check for it throughout

Reply:
Also the batch/Batch thing

Reply:
Fixed throughout